### PR TITLE
Old apple clone

### PR DESCRIPTION
**Pull request by Davidosky007**
#The Old Apple Website Clone

In this project we:

*we made use of background images

### CODE REVIEW

**Comment by Davidosky007** [comment]
> #The Old Apple Website Clone
> 
> In this project we:
> 
> *we made use of background images
*we made use of Gradients
*added a favicon for the page title

File: index.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+                <i class="fas fa-search fa-sm"></i>
+              </form>
+            </li>
+
+          </ul>
+        </nav>
+        <!-- Header section -->
+        <div class="main-text">
+          <h1 class="heading-1">What will your verse be? </h1>
+          <h2 class="heading-2">
```
Comment (umairarshadbutt) [inline index.html]: h1 and h2 should be a link

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @Naqhids and @davidosky007
Such a remarkable project you made :+1: ! Awesome README :100: !
We, however, do have some few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [ ] Header content should be linked like Original webpage
- [ ] Heading content is overlapping on the navbar (By minimizing the screen)
##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:
## :wave: :wave: Hello Team!! Great effort on the project!!

:100: Great that you use media queries even if they were not required.

## Required Changes
### [Mandatory] :red_circle: 

- [ ] MIT license link takes to a Linkedin profile. It should take to the actual license.

- [ ] Website is not displaying properly
image

- [ ] Navbar link elements are not visible enough. 
image

- [ ] Navbar should be inside the ```<header>``` tag. We should move it outside the ```<main>``` tag.

- [ ] Footer should be outside the ```<main>``` tag.

- [ ] All images should have an alternate text. We should fill the ```alt=" "``` attribute, never leave it blank.

### [OPTIONAL]  :large_blue_circle: 

- [ ] The project should follow this structure:
```
/
  css/
    style.css
  img/
    ... some images...
  js/
    ... some js scripts...
  index.html
```

### [Comments] :green_circle: 

- [ ] After the changes have been made do nor forget to update the live preview link in the README

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

## :wave: :wave:

## Required changes:
### [Mandatory] :red_circle: 

- [ ] Remove the License section from the README if there is no license.
image

- [ ] Previously required changes haven't been applied. We should review them and ask for a re-review until we have addressed all the issues.

> # Status: Changes Required 
> ##   Hello Team!! Great effort on the project!!
>  Great that you use media queries even if they were not required.
> 
> ## Required Changes
> ### [Mandatory] 
> * [ ]  MIT license link takes to a Linkedin profile. It should take to the actual license.
> * [ ]  Website is not displaying properly
>   image
> * [ ]  Navbar link elements are not visible enough.
>   image
> * [ ]  Navbar should be inside the `<header>` tag. We should move it outside the `<main>` tag.
> * [ ]  Footer should be outside the `<main>` tag.
> * [ ]  All images should have an alternate text. We should fill the `alt=" "` attribute, never leave it blank.
> 
> ### [OPTIONAL] 
> * [ ]  The project should follow this structure:
> 
> ```
> /
>   css/
>     style.css
>   img/
>     ... some images...
>   js/
>     ... some js scripts...
>   index.html
> ```
> 
> ### [Comments] 
> * [ ]  After the changes have been made do nor forget to update the live preview link in the README
> 
> ## Keep up the great work  and Happy Coding  !!

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Pull request closed without merging**